d3d9-nine/dri2.c

```diff
@@ -711,6 +714,37 @@ static BOOL dri2_probe(Display *dpy)
     struct dri_backend_priv *priv;
     BOOL res;
 
+    xcb_connection_t *conn = XGetXCBConnection(dpy);
+    xcb_dri2_query_version_cookie_t dri2_cookie;
+    xcb_dri2_query_version_reply_t *dri2_reply;
+    xcb_generic_error_t *error;
+    const xcb_query_extension_reply_t *extension;
+    const int major = 1;
+    const int minor = 4; //FIXME
```

Comment:
What's the FIXME about? We're not sure which version we require?

Reply:
correct. I haven't checked which one is the minimum we need.

Reply:
we only need 1.0, but must use 1.4 to get the version the xserver supports.